Detectors/TOF/workflow/src/CompressedDecodingTask.cxx

```diff
@@ -82,16 +91,18 @@ void CompressedDecodingTask::postData(ProcessingContext& pc)
   int n_orbits = n_tof_window / 3;
   int digit_size = alldigits->size();
 
-  LOG(INFO) << "TOF: N tof window decoded = " << n_tof_window << "(orbits = " << n_orbits << ") with " << digit_size << " digits";
+  // LOG(INFO) << "TOF: N tof window decoded = " << n_tof_window << "(orbits = " << n_orbits << ") with " << digit_size << " digits";
 
   // add digits in the output snapshot
   pc.outputs().snapshot(Output{o2::header::gDataOriginTOF, "DIGITS", 0, Lifetime::Timeframe}, *alldigits);
   pc.outputs().snapshot(Output{o2::header::gDataOriginTOF, "READOUTWINDOW", 0, Lifetime::Timeframe}, *row);
 
-  // RS FIXME: At the moment dummy output to comply with CTF encoded inputs
-  std::vector<uint32_t> patterns;
+  std::vector<uint32_t> patterns = mDecoder.getPatterns();
```

Comment:
should not be here ``std::vector<uint32_t>& patterns = mDecoder.getPatterns();`` ?

Reply:
Yes. Done